content/60.methods.md

```diff
@@ -106,8 +106,12 @@ Due to the needs of this project, project contributors also implemented new feat
 Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations.
 These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].
 
-Additionally, the fast-moving nature of the infodemic has led to a number of retractions and corrections of COVID-19 literature.
-<!--To Do: Describe integration with scite -- talk to Vince-->  
+A new plugin was also added to Manubot to support "smart citations".
+The plugin uses the [Scite](https://scite.ai/) service to display a badge below each citation with a set of icons and numbers indicating how many times that source has been mentioned, supported, or disputed.
+The scite badges were enabled in the HTML build of the document.
+The badge also indicates important editorial notices, such as retractions and corrections.
+This allows readers to roughly evaluate the reliability of cited sources at a glance.
```

Comment:
```suggestion
This alerts readers to major reliability issues of cited sources at a glance.
```

I wouldn't want to give the impression that retractions are the main way to evaluate reliability, especially in COVID literature.

Reply:
The "this" in that sentence was supposed to refer to the badge as a whole, not the retractions, but I didn't realize that Halie's edits made it refer to the retractions. Pushing a re-write soon.